### PR TITLE
playbook setup to prepare xcode installation via ansible

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [x ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/xcode-setup/files/install_brew.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+(
+	echo
+	echo "---> [install] Homebrew"
+	echo '' | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+	res_code=$?
+	echo "---> [install] Homebrew [done]"
+
+	exit $res_code
+) 2>> ~/Desktop/debug.log 1>> ~/Desktop/debug.log
```
Comment (bitrise-tom) [inline roles/xcode-setup/files/install_brew.sh]: Would it make more sense to name this log file according to what it is doing?
~/Desktop/brew_install.log ?

Comment (bitrise-tom) [reply roles/xcode-setup/files/install_brew.sh]: i see this:
roles/intermediate-setup/tasks/main.yml:# Setup the debug log file for custom scripts

File: roles/xcode-setup/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# Install brew ruby
+# `brew install ruby`
+- name: Install brew ruby
+  homebrew: name=ruby@{{ default_ruby }} state=present
+
+# Add ruby to path
+- name: Export brew ruby to PATH
+  shell: bash -l -c "export PATH=/usr/local/opt/ruby/bin:$PATH && export PATH=/usr/local/lib/ruby/gems/{{ default_ruby }}/bin:$PATH"
+
+# Install deps // fastlane xcode-install
```
Comment (bitrise-tom) [inline roles/xcode-setup/tasks/main.yml]: https://docs.ansible.com/ansible/latest/modules/gem_module.html

Can you use the Gem module here?

File: roles/xcode-setup/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# Install Homebrew
+# This needs a script to run, or directly by curl
+- name: run brew installer script
+  script: install_brew.sh creates=/usr/local/bin/brew
+- name: brew update
+  homebrew: update_homebrew=yes upgrade_all=no
+
+# Install brew ruby
+# `brew install ruby`
+- name: Install brew ruby
```
Comment (ghaabor) [inline roles/xcode-setup/tasks/main.yml]: Please use rbenv here and install the ruby using that

Comment (fehersanyi-bitrise) [reply roles/xcode-setup/tasks/main.yml]: aye aye sir

File: roles/xcode-setup/tasks/main.yml (hunk 2 above)
Comment (fehersanyi-bitrise) [reply roles/xcode-setup/tasks/main.yml]: oh nice catch I will set it up like that

**Pull request merged**